aedttest/aedt_test_runner.py

```diff
@@ -914,6 +914,7 @@ def execute_aedt(
 
     """
     aedt_path = get_aedt_executable_path(version)
+    log_path = os.path.splitext(project_path)[0] + ".log"
```

Comment:
basically, 
project_path could be optional according to the type
however, `os.path.splitext` does not support `None`